systemtest/src/test/java/io/strimzi/systemtest/components/bridge/HttpBridgeCors.java

```diff
@@ -2,7 +2,7 @@
  * Copyright Strimzi authors.
  * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
  */
-package io.strimzi.systemtest.bridge;
+package io.strimzi.systemtest.components.bridge;
```

Comment:
Not sure about the package `components` , I'm not sure that the previous structure wasn't right (for atleast this set of tests) - what is the components package doing here?
Aren't all the STs basically testing components?

Reply:
Yes that's true. I was just thinking to separate it somehow from other because tests in `bridge` focus mostly on the functionality of Bridge for example